TRENZ.Lib.RazorMail.SystemNet/Extensions/SystemNetMailMessageExtensions.cs

```diff
@@ -46,8 +48,12 @@ private static void SetMailHeaders(RazorMailMessage razorMessage, SystemNetMailM
 
         foreach (var item in razorMessage.Headers.ReplyTo)
             systemNetMessage.ReplyToList.Add(item.ToMailAddress());
+        
+        systemNetMessage.Headers["Importance"] = razorMessage.Headers.Importance.ToImportanceHeaderValue();
```

Comment:
Does `System.Net.Mail` expose any well-known header keys?

Reply:
It does, but from what I gather, its implementation is wrong. We _could_ set something like:

```csharp
systemNetMessage.Priority = MailPriority.High;
```

But [this also sets Priority](https://source.dot.net/#System.Net.Mail/System/Net/Mail/MailPriority.cs,442), which is I believe incorrect. So instead, we set them at a lower level.

Also, while `MailHeaderID` exists (sort of a `WellKnownHeaders`), it's `internal`…

Reply:
Ok, we checked and it's the best we can do...